### PR TITLE
LPTOCPCI-30: Camel K operator onboarding

### PR DESCRIPTION
**Pull request by chetna14manku**
Link to the ticket: [LPTOCPCI-30](https://issues.redhat.com/browse/LPTOCPCI-30)

### CODE REVIEW

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-jboss-fuse-camel-k-test-container-main-camel-k-ocp4.14-lp-interop-camel-k-interop-aws

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-jboss-fuse-camel-k-test-container-main-camel-k-ocp4.14-lp-interop-camel-k-interop-aws

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-chetna14manku-camel-k-test-container-debug-camel-k-ocp4.14-lp-interop-camel-k-interop-aws

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-jboss-fuse-camel-k-test-container-main-camel-k-ocp4.14-lp-interop-camel-k-interop-aws

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-chetna14manku-camel-k-test-container-debug-camel-k-ocp4.14-lp-interop-camel-k-interop-aws

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-jboss-fuse-camel-k-test-container-main-camel-k-ocp4.14-lp-interop-camel-k-interop-aws

**Comment by chetna14manku** [comment]
Latest job link: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/40565/rehearse-40565-periodic-ci-jboss-fuse-camel-k-test-container-main-camel-k-ocp4.14-lp-interop-camel-k-interop-aws/1674045634363002880

File: ci-operator/config/jboss-fuse/camel-k-test-container/jboss-fuse-camel-k-test-container-main__camel-k-ocp4.14-lp-interop.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"},
+          {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"},
+          {"step": "gather-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"},
+          {"step": "openshift-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"},
+          {"step": "camel-k-interop-tests", "failure_type": "all", "classification": "Test Failure", "jira_project": "CMLK", "jira_epic": "LPTOCPCI-290"}
```
Comment (calebevans) [inline ci-operator/config/jboss-fuse/camel-k-test-container/jboss-fuse-camel-k-test-container-main__camel-k-ocp4.14-lp-interop.yaml:41]: Can we add a line specifically for test failures to be sent to PQE's Jira project?

**Review by calebevans — COMMENTED** [review]
Only really saw one thing in the Firewatch config to possibly change.

File: ci-operator/config/jboss-fuse/camel-k-test-container/jboss-fuse-camel-k-test-container-main__camel-k-ocp4.14-lp-interop.yaml (hunk 1 above)
Comment (chetna14manku) [reply ci-operator/config/jboss-fuse/camel-k-test-container/jboss-fuse-camel-k-test-container-main__camel-k-ocp4.14-lp-interop.yaml]: Done

**Pull request merged**